basic/basic-01-basic-connector/README.md

```diff
@@ -25,6 +25,11 @@ with that we can build and run the connector from the root directory:
 java -jar basic/basic-01-basic-connector/build/libs/basic-connector.jar
 ```
 
+```windows command prompt
```

Comment:
The type after "```" helps markdown to render the document and is not shown to the user, as you can see:
![image](https://user-images.githubusercontent.com/72392527/224051247-22e2e7f8-f768-4f39-b252-441fd6345310.png)

I would therefore suggest the following
```suggestion
```bash
# for Windows users
```

This way, the promt is rendered correctly and readers know that this responds to Windows users.